google/cloud/functions/CMakeLists.txt

```
@@ -80,7 +81,8 @@ target_include_directories(functions_framework_cpp SYSTEM
                            PUBLIC $<INSTALL_INTERFACE:include>)
 target_link_libraries(
     functions_framework_cpp PUBLIC absl::time Boost::headers
-                                   Boost::program_options Threads::Threads)
+                                   Boost::program_options Threads::Threads
+                                   nlohmann_json::nlohmann_json)
```

As you probably noticed, the robots are asking for different formatting here.

```suggestion
    functions_framework_cpp
    PUBLIC absl::time Boost::headers Boost::program_options Threads::Threads
           nlohmann_json::nlohmann_json)
```